lib/fpm/command.rb

```diff
@@ -213,6 +213,13 @@ def help(*args)
     File.expand_path(val) # Get the full path to the script
   end # --before-upgrade
 
+  option "--triggered", "FILE",
```

Comment:
Trying to figure out if this flag is best put in `package/deb.rb` instead of this file. Some questions to think about?
Are triggers common to multiple package systems? Or just Debian?

If they're common, is the implementation common-enough across multiple package systems? That is, would the same flag work for both Debian and rpm packages?

Reply:
Hm, looks like rpm has something too: https://jfearn.fedorapeople.org/en-US/RPM/4/html/RPM_Guide/ch05s03.html

Not sure if it works the same way.